models/api_rate.go

```diff
@@ -63,17 +63,20 @@ func (r *Rate) SpecialServicesRequested() *SpecialServicesRequested {
 }
 
 func (r *Rate) Weight() Weight {
-	commoditiesWeight := r.Commodities.Weight()
-	if !commoditiesWeight.IsZero() {
-		// Assume the weight must be between than 13 and 150 lbs.
-		// If the weight is less than 13 lbs, assume a weight of 13 lbs, which is
-		// heavy enough that the destination will matter when choosing between two
-		// fedex ground rates
-		commoditiesWeight.Value = math.Min(commoditiesWeight.Value, 150.0)
-		return commoditiesWeight
+	commoditiesSumWeight := r.Commodities.Weight()
+
+	if !commoditiesSumWeight.IsZero() {
+		var maximumWeightInLbs = 150.0
+
+		var maxWeight = math.Min(commoditiesSumWeight.Value, maximumWeightInLbs)
+		commoditiesSumWeight.Value = maxWeight
+
+		return commoditiesSumWeight
 	}
 
-	return Weight{Units: WeightUnitsLB, Value: 13}
+	var safeGuardForZeroWeightLbs = 0.9375
+
+	return Weight{Units: WeightUnitsLB, Value: safeGuardForZeroWeightLbs}
```

Comment:
curious are we not allowed to use unitOZ?

Reply:
I'm worried because of the services that consume this API, I think it would be prudent to keep it in its current format.